README.md

```diff
@@ -2,16 +2,19 @@
 
 # Project Name
 
-> Description the project.
+> One paragraph statement about the project.
 
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
 
 ## Built With
 
 - Major languages
 - Frameworks
 - Technologies used
 
-## Live Demo (if available)
+## Live Demo
 
 [Live Demo Link](https://livedemo.com)
```

Comment:
Kindly add the link to the live demo if it's available, if not you need to remove this part in your README file
Below this line also you need to remove unnecessaries parts of lines of this document and add also your contact info and acknowledgments.
Bref you need to update all your README file :smile: